test/playwright/playwright.config.ts

```diff
@@ -1,7 +1,15 @@
+import path from 'path'
+
+import { addAliases } from 'module-alias'
+
 import type { PlaywrightTestConfig } from '@playwright/test'
 
+addAliases({
+  '~': path.resolve(process.cwd(), 'src'),
+  '~~': process.cwd(),
```

Comment:
Nice!

Reply:
Nuxt already uses this alias to point to `<rootDir>`. Can we pick something else here?

Reply:
It is pointing to the root dir by using `process.cwd()`, these aliases match nuxt and ts configuration.